code/modules/mob/living/living.dm

```diff
@@ -286,6 +286,8 @@
 
 	if(istype(AM) && Adjacent(AM))
 		start_pulling(AM)
+	else if(bloodcrawl == BLOODCRAWL_EAT) //so demons can ctrl + click tiles to teleport to them
```

Comment:
Mark your changes ![like this](https://github.com/yogstation13/tg-rebase/blob/master/yogstation/README.md)